state/state.go

```diff
@@ -7,9 +7,18 @@ import (
 	"context"
 
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/ava-labs/avalanchego/x/merkledb"
 )
 
+const (
+	Read  PermissionBit = 0
+	Write PermissionBit = 1
+	// TODO: Handle Allocate permission
+	Allocate      PermissionBit = 2
```

Comment:
nit: let's put this above `Write` (make it =1)

Reply:
Id did the same with dimension definition: https://github.com/ava-labs/hypersdk/blob/4bd6d720a5bc8ab9c6361e05f3c00d9c4780720d/chain/fee_manager.go#L20-L22